ROMFS/px4fmu_common/init.d/airframes/17002_TF-AutoG2

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+#
+# @name TF:AutoG2
+#
+# @type Autogyro
+# @class Autogyro
+#
+# @output MAIN1 rotor_head_L
+# @output MAIN2 rotor_head_R
+# @output MAIN3 elevator
+# @output MAIN4 rudder
+# @output MAIN5 rudder (second, optional)
+# @output MAIN6 throttle
+# @output MAIN7 wheel
+#
+# @output AUX1 prerotator (optional)
```

Comment:
Based on the current config, there is no AUX mixer loaded, or do I miss something?

Reply:
Aux pass mixer was added in new commit: https://github.com/PX4/Firmware/pull/14015/commits/dc357dc4147f4b12b62fd38b027ec31cd6c23ca9

Reply:
Then I'd rename this to `feed-through of RC AUX1 channel`

Reply:
Ok, description is chaneged

https://github.com/PX4/Firmware/blob/76b3ff04598f5a4a4a4e1487a93386d718cf4643/ROMFS/px4fmu_common/init.d/airframes/17002_TF-AutoG2#L16-L17